<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>convert_dialog</class>
 <widget class="QDialog" name="convert_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Преобразование</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="src_button">
           <property name="text">
            <string>Исходный файл</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="dst_button">
           <property name="text">
            <string>Результирующий файл</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="src_edit">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dst_edit">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="convert_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Преобразовать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>src_button</tabstop>
  <tabstop>dst_button</tabstop>
  <tabstop>convert_button</tabstop>
  <tabstop>src_edit</tabstop>
  <tabstop>dst_edit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
